src/lib/Sympa/Regexps.pm

```diff
@@ -47,6 +47,8 @@ use constant email =>
 use constant family_name => qr'[a-z0-9][a-z0-9\-\.\+_]*';
 ## Allow \s for template names
 use constant template_name => qr'[a-zA-Z0-9][a-zA-Z0-9\-\.\+_\s]*';
+# cf. RFC5322, 2.2.
+use constant header_field_name => qr'[!-9;-~]+';
```

Comment:
I’m not familiar with the interval `!-9`, but I guess it contains a lot of ascii chars and no accentuated letters.

I trust your regex skills :slightly_smiling_face:

Reply:
`[!-9;-~]` represents "printable ASCII character except colon".  I only hope I have not betrayed your trust 😓 .

Reply:
You didn’t! :ok_hand: